doc/additional_commands.rst

```diff
@@ -0,0 +1,100 @@
+Universum modes
+===============
+
+Along with default mode Universum provides the following useful additional commands:
+
+* `init <additional_commandst#init>`_
+* `run <additional_commandst#run>`_
+* `poll <additional_commandst#poll>`_
+* `submit <additional_commandst#submit>`_
+* `github-handler <additional_commandst#github-handler>`_
+
+
+.. _additional_commandst#init:
+
+Initialize Universum
+--------------------
+
+If you have Universum installed via ``{pip} install -U universum`` or downloaded it from GitHub and changed
```

Comment:
It would be nice to add similar description for poll and other commands

Reply:
Yes, it totally would.

Reply:
Please add them